styles.css

```diff
@@ -422,7 +523,38 @@ textarea {
   color: #fff;
 }
 
+.btn-flex {
+  display: flex;
+}
+
+.btn-flex button:first-child {
+  margin-right: 12px;
+}
+
+.live_link,
+.source_link {
+  text-decoration: none !important;
+}
+
 @media (min-width: 768px) {
+  div.modal {
+    width: 80%;
```

Comment:
- [ ] Kindly decrease the modal's width to `65%` instead of `80%` to make your design match the Figma design.